config/environments/production.rb

```diff
@@ -44,9 +44,19 @@
   # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
   # config.force_ssl = true
 
-  # Use the lowest log level to ensure availability of diagnostic information
-  # when problems arise.
-  config.log_level = :debug
+  config.log_level = :info
+  config.lograge.enabled = true
+
+  config.lograge.custom_options = lambda do |event|
+    params = event.payload[:params].reject do |k|
+      ['controller', 'action'].include? k
+    end
+    log_hash = {"params"=> params, "time"=>event.time}
+    if not event.payload[:exception].blank?
```

Comment:
1) don't use ruby `not`, use `!` ([source](https://github.com/bbatsov/ruby-style-guide#bang-not-not)) (basically cause it doesn't have the same binding priority)
2) here just use `if event.payload[:exception].present?`